edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/TwinConfigSource.cs

```diff
@@ -248,7 +252,7 @@ public static bool ExtractHubTwinAndVerify(TwinCollection twinDesiredProperties)
                 JToken integrity = twinJobject["integrity"];
                 JToken header = integrity["header"];
                 JToken signerCertJtoken = integrity["header"]["signercert"];
-                string combinedCert = signerCertJtoken.First.ToString() + signerCertJtoken.Last.ToString();
+                string combinedCert = signerCertJtoken[0].ToString() + signerCertJtoken[1].ToString();
```

Comment:
Would it make sense to make the assembly the array generically?  Like using Enumerable.Aggregate()?